src/index.html

```diff
@@ -13,20 +13,30 @@
     <title>Document</title>
     <link
       rel="stylesheet"
-      href="style.css"
+      href="/src/style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
+    <form
+      class="search-container"
+      action="#"
+      method="post"
       data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    >
+      <div class="search-field large">
+        <input
+          type="text"
+          placeholder="Try “Los Angeles“"
+          data-qa="big"
+        />
+      </div>
+      <div class="search-field small">
+        <input
+          type="text"
+          data-qa="small"
+          placeholder="Try “Los Angeles“"
+        />
```

Comment:
Remember about correct indentation between parent and child elements. The `<input>` tag should be indented two spaces from the `<div>` tag.